### PR TITLE
Rewrite based on Aerys

### PR DESCRIPTION
**Pull request by kelunik**
See https://github.com/php-pm/php-pm/pull/267.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,11 +3,17 @@
...
-        "php-pm/php-pm": "dev-master"
+        "php-pm/php-pm": "dev-amp as dev-master"
     },
     "autoload": {
         "psr-4": {
             "PHPPM\\": ""
         }
-    }
+    },
+    "repositories": [
```
Comment (staabm) [inline composer.json]: open todo which needs to be fixed when the corresponding php-pm PR landed.

File: Bridges/HttpKernel.php [hunk 2]
```diff
@@ -191,23 +181,19 @@ protected function mapResponse(HttpResponse $reactResponse, SymfonyResponse $syR
...
+                $nativeHeaders[$name][] = $value;
             }
         }
 
         // after reading all headers we need to reset it, so next request
         // operates on a clean header.
         header_remove();
 
-        $headers = array_merge($nativeHeaders, $syResponse->headers->allPreserveCase());
+        $headers = array_merge(
```
Comment (staabm) [inline Bridges/HttpKernel.php]: lowering the keys might be a bc break ?

Comment (kelunik) [reply Bridges/HttpKernel.php]: Where exactly do you see the BC break? It's just outputting of the headers, and Aerys lowercases them anyway.

Comment (staabm) [reply Bridges/HttpKernel.php]: After a more in depth look I couldnt find a actual problem

**Comment by dzubchik** [comment]
Should it be closed because of this https://github.com/php-pm/php-pm/pull/267?

**Pull request closed without merging**